docs/source/tagset/elemento-year.rst

```diff
@@ -3,18 +3,22 @@
 <year>
 ^^^^^^
 
-Aparece em
-  :ref:`elemento-pub-date`, :ref:`elemento-product`, 
+Aparece em:
+
+  :ref:`elemento-pub-date`
+  :ref:`elemento-product`
   :ref:`elemento-element-citation`
```

Comment:
Corrigido. Fui verificar se em ``<day>`` e em ``<month>`` estava definido e encontrei:
* em ``<day>`` - consta como  `` :ref:`elemento-history` `` - alterado para `` :ref:`elemento-date` ``;
* em ``<month>`` - incluído `` :ref:`elemento-date` ``

Ainda em ``<day>`` sob *Aparece em* há ocorrência de:
 `` :ref:`elemento-date-in-citation` ``

Se este elemento existe, não deveria ser definido também para ``<year>`` e ``<month>``?